pyiron/atomistics/structure/atoms.py

```diff
@@ -2389,6 +2389,10 @@ def symmetrize_vectors(
         if vectors.shape != self.positions.shape:
             print(vectors.shape, self.positions.shape)
             raise ValueError('Vector must be a natom x 3 array: {} != {}'.format(vectors.shape, self.positions.shape))
+        try:
+            self._symmetry_dataset
+        except AttributeError:
+            self._symmetry_dataset = None
```

Comment:
Can not we just set `self._symmetry_dataset = None` in the init function?

Reply:
That's what I do, but somehow when an old job was reloaded, it didn't have the attribute.

Reply:
That is strange - maybe the notebook was not restarted?

Reply:
If the notebook is not restarted it shouldn't contain the function in the first place

Reply:
I am not sure what happens in the notebook internally when we reload a job, but when an attribute is defined in the `__init__()` function then it can not cause an `AttributeError`. The exception only makes the code hard to read and the next person has no chance to understand why it was introduced in the first place.

Reply:
Apparently it matters where the attribute is defined. I don't know why...

Reply:
Anyway I could remove try and error